highfidelity/hifi
# 7991 Change Instances of Edit Mode to Create Mode

**Test plan**

1. Go to application's 'edit' menu
2. Ensure 'Show Lights and Particle Systems in Create Mode' and 'Show Zones in Create Mode' menu items are shown instead of 'Show Lights and Particle Systems in Edit Mode' and 'Show Zones in Edit Mode'
3. Go to tablet's help
4. Ensure all 4 help images have 'create' instead of 'edit' at the bottom right area, like on the screenshot below: 

![image](https://user-images.githubusercontent.com/23638/33287091-99ca4346-d3c8-11e7-9a6a-2e5682fb2700.png)


https://highfidelity.manuscript.com/f/cases/7991/Change-Instances-of-Edit-Mode-to-Create-Mode
